easybuild/easyconfigs/c/CP2K/CP2K-svn-CrayGNU-2016.11-cuda-8.0.54.eb

```diff
@@ -0,0 +1,59 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
+
+name = 'CP2K'
+version = "svn"
```

Comment:
how about:
version = "2017.02"? 

Unless you don't plan to make it public (in that case you can leave it like this).